symphony-app-authentication-fe/sagas/apiCalls.js

```diff
@@ -0,0 +1,39 @@
+import axios from 'axios';
+
+const baseAuthenticationUrl = "";
+
+export default class ApiCalls {
+  constructor(baseAuthenticationUrl) {
+    this.baseAuthenticationUrl = `${baseAuthenticationUrl}/v1/application/jwt`
```

Comment:
this.baseAuthenticationUrl = `${baseAuthenticationUrl}/v1/application`

Reply:
Done.